### DISCUSSION TITLE
New DS223j and Club3D 2.5Gigabit Adapter are working fine!

### DISCUSSION THREAD

**Discussion by Kamika242** [bb-qq/r8152#326] [Show and tell]
Just wanted to tell that the new Synology DS223j (DSM 7.2 Update 3) and the Club3D 2.5 Gigabit USB adapter (CAC-1420) are working fine with driver package r8152-rtd1619b-2.17.1-1_7.2.spk. Only the LAN-LED of the Synology doesn't work when this connection is being used.

Regards and a big "thank you" to bb-qq,

Kamika242

**Comment by jyubonk** [comment]
Think DS423+ also can use this code?

https://www.asus.com/my/networking-iot-servers/wired-networking/wired-adapters/usb-c2500/

Will this asus adapters works with this code on DS423+?

**Reply by Kamika242** [reply]
Don't know, it depends on the chipset of the Asus adapter, it has to be Realtek r8152, r8153 or r8156(?) based. I took the Club3D adapter, cause it's listed under supported devices. The DS423+ uses the Intel Celeron J4125 chip, so you need another driver from the list, if available.

Edit: This should be the correct driver for the DS423+: r8152-geminilake-2.17.1-1_7.2.spk, assuming you are running DSM 7.2.

**Reply by jyubonk** [reply]
Tested and functioning well.

ASUS USB-C2500

![image](https://github.com/bb-qq/r8152/assets/76234190/cf4e4376-b892-4508-a0a7-9c0896f6dce6)

![image](https://github.com/bb-qq/r8152/assets/76234190/8b5dbac2-22ec-490e-a0ed-9269c1824633)

Thank you so much @Kamika242!!

**Comment by ORah82** [comment]
Hey I have the DS223 running DSM 7.2.1 Update 5. So the driver package should work for me too. Do you have the link to the package you installed? Thanks!

**Reply by Kamika242** [reply]
https://github.com/bb-qq/r8152/releases/download/2.18.1-1/r8152-rtd1619b-2.18.1-1_7.2.spk

**Reply by ORah82** [reply]
Thanks! ✌️✌️

**Reply by ORah82** [reply]
Hey I've followed all the steps and have the same ethernet adapter your recommended. Installation went fine and driver is running but when I plug the USB-A plug in it doesn't recognize it thus no connection. I've tried all 3 ports. Any tips?

**Reply by ORah82** [reply]
NM I got it going. 😅

Problem though it's not recognizing the adapter: 
<img width="622" alt="Screenshot 2024-10-03 225835" src="https://github.com/user-attachments/assets/2802dea3-5d09-41d7-a08e-83f9e69fc094">
![Screenshot 2024-10-03 230014](https://github.com/user-attachments/assets/e6e1c6a3-2040-4245-925e-22ddb9816aac)

**Reply by Kamika242** [reply]
The adapter worked for me from the beginning on.
Did you restart the Syno? I'm sorry, I can't help much more, due to I sold my Syno, it made to much problems and finally it was much too oversized for my purposes.
